python/fledge/services/core/api/support.py

```diff
@@ -208,14 +204,14 @@ async def get_syslog_entries(request):
             # cmd = non_total_template.format(valid_source[source], _SYSLOG_FILE, offset, limit)
             pattern = '({})\[.*\].*{}:'.format(valid_source[source], levels)
             cmd = '{} -offset {} -limit {} -pattern \'{}\' -logfile {} -source {} -level {}'.format(scriptPath, offset, limit, pattern, _SYSLOG_FILE, source, level)
-            _logger.info('********* non_totals=true: new shell command: {}'.format(cmd))
+            _logger.debug('********* non_totals=true: new shell command: {}'.format(cmd))
 
         t1 = datetime.datetime.now()
-        a = subprocess.Popen([cmd], shell=True, stdout=subprocess.PIPE).stdout.readlines()
+        rv = subprocess.Popen([cmd], shell=True, stdout=subprocess.PIPE).stdout.readlines()
         t2 = datetime.datetime.now()
-        c = [b.decode() for b in a]  # Since "a" contains return value in bytes, convert it to string
-        _logger.info('********* Time taken for grep/tail/head subprocess: {} msec'.format((t2 - t1).total_seconds()*1000))
-        response['logs'] = c
+        rv_str = [b.decode() for b in a]  # Since "a" contains return value in bytes, convert it to string
```

Comment:
Here is the reason for the unit test failure on this commit is due to incomplete variable refactoring. See code snippet
```
[b.decode() for b in a] 
```
reference of `a` is still there. It should be with `rv`

Reply:
yeah, thanks for pointing out.